.github/workflows/compatibility.yml

```diff
@@ -55,7 +55,7 @@ jobs:
         woocommerce: [ 'beta' ]
         wordpress:   [ 'latest' ]
         gutenberg:   [ 'latest' ]
-        php:         [ '7.1', '8.0', '8.1' ]
+        php:         [ '7.2', '8.0', '8.1' ]
```

Comment:
Since Woo 6.5 (or currently it's the beta version), Woo raises the PHP minimum requirement to PHP 7.2. 

Ref https://github.com/woocommerce/woocommerce/blob/a784e6274cdedf12050bdbee3c839eb502700266/plugins/woocommerce/woocommerce.php#L12